hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/WordCount.java

```diff
@@ -131,11 +145,14 @@ public static void main(String[] args) throws Exception {
         job.setInputFormatClass(PravegaInputFormat.class);
 
         FileInputFormat.addInputPath(job, new Path(otherArgs[0]));
-        Path outputpath = new Path(otherArgs[4]);
+        Path outputpath = new Path(out);
         FileOutputFormat.setOutputPath(job, outputpath);
 
         boolean result = job.waitForCompletion(true);
         readAndPrint(outputpath, conf);
+
+        System.out.printf("End positions of stream cut %s/%s: '%s'\n", scope, stream, endPos);
```

Comment:
Consider use logging for these purposes.

Reply:
accepted.